lib/notify/notification.rb

```diff
@@ -2,6 +2,10 @@
 
 module Notification
   INVITE_TEMPLATE = "afdb4827-0f71-4588-b35d-80bd514f5bdb".freeze
+  REMINDER_TEMPLATE = "bbc34127-7fca-4d78-a95b-703da58e15ce".freeze
+
+  REMINDER_TEMPLATE_STANDARD_SUBJECT = "Your GOV.UK Verify certificates will expire on %s".freeze
```

Comment:
Did not know you could pass values in with `%s` and `%d`, quite cool

Reply:
Just a question what happens after all these reminders. And still no rotation has been done, I know this would be no longer a reminder. Am just curious

Reply:
the service goes down, and that's a P2 for second line... but it as well may be that they added a new signing cert, but forgot to remove the old one.. so it would still work...